src/QueryDescription/index.js

```diff
@@ -163,10 +163,14 @@ export function between(left: number, right: number): Comparison {
   return { operator: 'between', right: { values: [left, right] } }
 }
 
-export function like(value: Value): Comparison {
+export function like(value: string): Comparison {
   return { operator: 'like', right: { value } }
 }
 
+export function sanitizeLikeString(value: string): string {
+  return value.replace(/[^a-zA-Z0-9]/g, '')
```

Comment:
I think replacing unwanted characters with `_` is a better option, otherwise if someone searches "Foo Bar!" it won't find a thing called precisely "Foo Bar!".